README.md

```diff
@@ -1,17 +1,80 @@
-## Documentation Guidelines
+# Documentation Guidelines
 
-### 🚀 Contributing
+## 🚀 Contributing
 
-### Before Proposing Additions
+## Before Proposing Additions
 
-### Style Guidelines
+## Style Guidelines
 
-#### Code Examples
+### Code Examples
 
-#### Notices
+### Notices
 
-#### Screenshots
+### Screenshots
 
-#### Terminology
+### Terminology
+Appwrite has many services and features with many ways to express the same concepts. To avoid confusion, this section suggests a standard set of terms used across Appwrite documentation to describe features and concepts.
+#### General
```

Comment:
Let's maybe add:
- Usage stats
- Audit logs

Reply:
I don't have any experience with these, what terms do you suggest under them?